rest/service/dashboardTemplate.go

```diff
@@ -160,6 +160,19 @@ var (
 				Title: "Advanced Cluster Security",
 			},
 		},
+		models.Subscriptions: models.ModuleFederationMetadata{
+			Scope:    "landing",
```

Comment:
The scope is connected to the appName's module scope. So for the subs inventory it should be `subscriptionInventory`
```suggestion
			Scope:    "subscriptionInventory",
```